module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,13 @@
+#include "sort.hpp"
+
+#include <set>
+
+std::deque<std::string> lengthSort(const std::forward_list<std::string>& words) {
+    std::multiset<std::string, decltype(compare)> ms(compare);
```

Comment:
Use std::multiset is not necessary. If you have comparator, just use std::sort. 
You are python programmer :)?

Reply:
Replaced with 'sort' method of forward_list (with lambda).